modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainer.java

```diff
@@ -120,48 +225,70 @@ public SELF withAdminPassword(final String adminPassword) {
      * @param database Automatically initializes a database with the name of this environment variable.
      * @return a reference to this container instance
      */
-    public SELF withDatabase(final String database) {
+    public InfluxDBContainer withDatabase(final String database) {
         this.database = database;
-        return self();
+        return this.self();
     }
 
     /**
-     * Set env variable `INFLUXDB_USER`.
+     * Set env variable `DOCKER_INFLUXDB_INIT_ORG`.
      *
-     * @param username The name of a user to be created with no privileges. If `INFLUXDB_DB` is set, this user will
-     *                 be granted read and write permissions for that database.
+     * @param organization The organization for the initial setup of influxDB.
      * @return a reference to this container instance
      */
-    public SELF withUsername(final String username) {
-        this.username = username;
-        return self();
+    public InfluxDBContainer withOrganization(final String organization) {
+        this.organization = organization;
+        return this;
     }
 
     /**
-     * Set env variable `INFLUXDB_USER_PASSWORD`.
+     * Set env variable `DOCKER_INFLUXDB_INIT_BUCKET`.
      *
-     * @param password The password for the user configured with `INFLUXDB_USER`. If this is unset, a random password
-     *                 is generated and printed to standard out.
+     * @param bucket Automatically initializes a bucket with the name of this environment variable.
      * @return a reference to this container instance
      */
-    public SELF withPassword(final String password) {
-        this.password = password;
-        return self();
+    public InfluxDBContainer withBucket(final String bucket) {
+        this.bucket = bucket;
+        return this;
     }
 
     /**
-     * @return a url to influxDb
+     * Set env variable `DOCKER_INFLUXDB_INIT_RETENTION`.
```

Comment:
```suggestion
     * Set the retention in days
```
Is this retention in days?